modules/apps/sharing/sharing-api/src/main/java/com/liferay/sharing/security/permission/SharingEntryAction.java

```diff
@@ -39,14 +39,17 @@ public enum SharingEntryAction {
 	public static Collection<SharingEntryAction> getSharingEntryActions(
 		long bitwiseValue) {
 
-		return Stream.of(
-			values()
-		).filter(
-			sharingEntryAction ->
-				(sharingEntryAction.getBitwiseValue() & bitwiseValue) != 0
-		).collect(
-			Collectors.toList()
-		);
+		return TransformUtil.transformToList(
```

Comment:
@jeffwu0724 To make the code simpler, use `ListUtil.filter(Arrays.asList(values()), sharingEntryAction -> ...)`.

Reply:
@jeffwu0724 `Arrays.asList()` creates an `Arrays`-private implementation of array based list, which directly wraps the passed array, so no copy is involved in this step. Thus, using this, you achieve the same, with less lines of code.